src/System.Private.ServiceModel/src/System.Private.ServiceModel.builds

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>
+    <Project Include="System.Private.ServiceModel.csproj" />
+	<Project Include="System.Private.ServiceModel.csproj">
+      <OSGroup>Linux</OSGroup>
+	  <TargetGroup>dnxcore50</TargetGroup>
```

Comment:
You should only ever list entries for different OSGroup's if you have something unique for those OSGroups. Otherwise if you are pure IL and don't have anything unique don't list any OSGroup and your project will be "AnyOS" and thus will be available on any OS.